### PR TITLE
Update xlf files

### PR DESCRIPTION
**Pull request by elinor-fung**
Follow-up for https://github.com/dotnet/runtime/pull/67860

### CODE REVIEW

**Review by jkoritzinsky — APPROVED** [review]

File: src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/Resources/xlf/Strings.de.xlf [hunk 1]
```diff
@@ -83,8 +83,8 @@
         <note />
       </trans-unit>
       <trans-unit id="ConfigurationNotSupportedMessageValue">
-        <source>The specified value '{0}' for '{1}' is not supported by source-generated P/Invokes. If the specified configuration is required, use a regular `DllImport` instead.</source>
-        <target state="translated">Der angegebene Wert \"{0}\" für \"{1}\" wird von quellgenerierten P/Invokes nicht unterstützt. Wenn die angegebene Konfiguration erforderlich ist, verwenden Sie stattdessen einen regulären \"DllImport\".</target>
+        <source>The specified value '{0}' for '{1}' is not supported by source-generated P/Invokes. If the specified value is required, use a regular `DllImport` instead.</source>
+        <target state="needs-review-translation">Der angegebene Wert \"{0}\" für \"{1}\" wird von quellgenerierten P/Invokes nicht unterstützt. Wenn die angegebene Konfiguration erforderlich ist, verwenden Sie stattdessen einen regulären \"DllImport\".</target>
```
Comment (gfoidl) [inline src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/Resources/xlf/Strings.de.xlf:87]:
```suggestion
        <target state="needs-review-translation">Der angegebene Wert \"{0}\" für \"{1}\" wird von code-generierten P/Invokes nicht unterstützt. Wenn der angegebene Wert erforderlich ist, sollte ein klassischer \"DllImport\" verwendet werden..</target>
```

File: src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/Resources/xlf/Strings.de.xlf [hunk 2]
```diff
@@ -208,8 +208,8 @@
         <note />
       </trans-unit>
       <trans-unit id="InTwoStageMarshallingRequiresToNativeValueMessage">
-        <source>The marshaller type '{0}' supports marshalling in the 'In' direction with the 'TwoStageMarshalling' feature must provide a 'ToNativeValue' instance method that returns the native value</source>
-        <target state="translated">Der Marshallertyp \"{0}\" unterstützt das Marshalling in der Richtung \"In\" mit der Funktion \"TwoStageMarshalling\" muss eine ToNativeValue-Instanzmethode bereitstellen, die den nativen Wert zurückgibt.</target>
+        <source>A marshaller type '{0}' that supports marshalling in the 'In' direction with the 'TwoStageMarshalling' feature must provide a 'ToNativeValue' instance method that returns the native value</source>
+        <target state="needs-review-translation">Der Marshallertyp \"{0}\" unterstützt das Marshalling in der Richtung \"In\" mit der Funktion \"TwoStageMarshalling\" muss eine ToNativeValue-Instanzmethode bereitstellen, die den nativen Wert zurückgibt.</target>
```
Comment (gfoidl) [inline src/libraries/System.Runtime.InteropServices/gen/LibraryImportGenerator/Resources/xlf/Strings.de.xlf:212]: This German sentence does not really make sense / it's translated sub-optimal.

**Review by gfoidl — COMMENTED** [review]
@elinor-fung I don't know if this is the right place here, but some translations could be improved. Is there a follow-up review for the translation (maybe an intern one) or should native speakers have a look at the translations?

I just looked in this PR by curiosity and spotted these two.

**Comment by elinor-fung** [comment]
Thanks, @gfoidl.

> Is there a follow-up review for the translation (maybe an intern one) or should native speakers have a look at the translations?

@cristianosuzuki77 - do you know what the process is here for collecting/incorporating feedback for translations? How should we go about getting @gfoidl's suggested improvements incorporated?

**Pull request merged**